### PR TITLE
Support PHPUnit 6 and PHPUnit Mock Objects 4

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,14 @@
 language: php
 
 php:
-  - 5.4
-  - 5.5
   - 5.6
   - 7.0
   - 7.1
   - hhvm
```
Comment (mnapoli) [inline .travis.yml]: Feel free to drop HHVM as well if it causes you any trouble.

Comment (carusogabriel) [reply .travis.yml]: Waiting for `Travis CI`'s answer :smile:

**Comment by carusogabriel** [comment]
@mnapoli Should we still support `PHPUnit 5`? They are causing some trouble

**Comment by mnapoli** [comment]
@gabriel-caruso no the older versions will work fine for PHPUnit 5, let's target v6 if it's easier.

**Comment by carusogabriel** [comment]
@mnapoli Did. I relied on `PHP-DI` Travis CI file for this one. Also, issue #6 can be close. As soon as you target a new version, I'll update `PHP-DI` for `PHPUnit 6`.

**Comment by mnapoli** [comment]
Awesome thanks, I'll tag a new release.

**Pull request merged**